examples/s3Node.ts

```diff
@@ -1,23 +1,8 @@
-import { S3Client, S3ClientConfig, GetObjectCommand, PutObjectCommand } from "@aws-sdk/client-s3";
-import { StoreInterface } from "./interface";
+// import { PassThroughCallback, StoreInterface } from "@dsnp/sdk/core/store";
```

Comment:
This is actually an example comment. It is showing how to import the StoreInterface from the *real* sdk. Not sure if it is needed however.

Reply:
If that's so then I think it needs some explanation, since it just looks like you accidentally left it in.  It's better to put it in maybe a README in this directory though, along with some other info maybe?